server/controllers/auth/login.js

```diff
@@ -0,0 +1,34 @@
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+
+const loginValidation = require('../../utils/validation/loginValidation');
+const applicant = require('../../database/models/applicant');
+require('env2')('config.env');
+
+const SECRET = process.env.SECRET_KEY;
+
+const login = (req, res) => {
+  const { email, password } = req.body;
+  loginValidation(email, password)
+    .then((valid) => {
+      if (!valid) throw Error;
```

Comment:
@ahmadysalah It might make the code simpler if you returned the 400 error on this line instead of throwing an error and then catching it on line 31.  Then the code flow reads top-to-bottom instead of jumping around.  Also, you could format the json response as an object with an error message instead of just a string argument:
```
if (!valid) res.status(400.json({ error: 'Please fill in all required fields' })
```

Reply:
thanks alot for your review.
i used this way because I have already catch with the same message, 
...
so I will change it thanks .